dpgen/generator/run.py

```diff
@@ -1295,7 +1426,13 @@ def _make_fp_vasp_inner (modd_path,
         cwd = os.getcwd()
         for ii in fp_tasks:
             os.chdir(ii)
-            dump_to_poscar('conf.dump', 'POSCAR', type_map)
+            if model_devi_engine == "lammps":
+                dump_to_poscar('conf.dump', 'POSCAR', type_map, fmt = "lammps/dump")
+            elif model_devi_engine == "gromacs":
+                # dump_to_poscar('conf.dump', 'POSCAR', type_map, fmt = "gromacs/gro")
+                dump_to_deepmd_raw('conf.dump', 'deepmd.raw', type_map, fmt = 'gromacs/gro')
```

Comment:
Is there any problem to use poscar instead of deepmd/raw? Or can we use deepmd/raw for lammps? It's better to use a single format for all engines, if there is no technical issues.

Reply:
BTW: I think the code here (not your PR) is also a mass.

Reply:
"Is there any problem to use poscar instead of deepmd/raw?" Yes this is problem. POSCAR format does not keep the original order, (VASP doesn't care about order). But the order should be kept in Gromacs (or Amber), such as the topology can work. 
I think probably we could use DeePMD-kit's format, but all other fp methods depend on POSCAR currently, and DeepMD-kit's files should have multiple files such as type.raw, coord.raw, etc, and is not convenient for users to export and visualize. So currently I don't think clearly.

Reply:
Ok. We may resolve this problem later.